dts/Kconfig

```diff
@@ -32,6 +32,13 @@ config HAS_DTS_SPI
 	  This option specifies that the target platform supports device tree
 	  configuration for SPI.
 
+config HAS_DTS_I2S
```

Comment:
Is that used finally?

Reply:
No, I'm removing it.
Give me 10 minutes and I will resubmit the patch

Reply:
@erwango 
done! I addressed all the points